sshtunnel.py

```diff
@@ -46,8 +46,6 @@
 TRACE_LEVEL = 1
 _CONNECTION_COUNTER = 1
 _LOCK = threading.Lock()
-#: Timeout (seconds) for the connection to the SSH gateway, ``None`` to disable
-SSH_TIMEOUT = None
```

Comment:
I know that someone uses code like so:
```
import sshtunnel
import threading
import time

# incompatible !
sshtunnel.SSH_TIMEOUT = 3

tunnel = sshtunnel.SSHTunnelForwarder(
            ('172.18.19.20', 22),
            ssh_username='user',
            ssh_password='password',
            remote_bind_address=('192.168.1.10', 24),
            set_keepalive=0.5
        )

```

It's already a part of a public API  :(

You can just use it as default for `gateway_timeout`

Reply:
@pahaz 

I hadn't considered that. Okay, that's fixable.

What do you think about the following?

1. Leave SSH_TIMEOUT where it is. 

```py
#: Timeout (seconds) for the connection to the SSH gateway, ``None`` to disable
SSH_TIMEOUT = None
```

2. In the constructor for SSHTunnelForwarder, set self.gateway_timeout based upon both the argument and the global value:

```py
self.gateway_timeout = gateway_timeout if gateway_timeout is not None else SSH_TIMEOUT
```

In the public API, there would be two ways to set the timeouts. One would be global, and the other would be for a single instance. If you set both timeouts, the setting on the specific instance is used.

Reply:
it's ok